tests/Security/Voter/CitizenProject/CreateCitizenProjectVoterTest.php

```diff
@@ -52,17 +52,9 @@ public function testNonAdherentIsNotGranted()
         $this->assertGrantedForAdherent(false, true, $adherent, CitizenProjectPermissions::CREATE);
     }
 
-    public function testAdherentIsNotGrantedWhenReferent()
```

Comment:
Please adapt and move the test below, naming it `testReferentIsGranted`

Reply:
Done, thx !